src/index.js

```diff
@@ -1,10 +1,12 @@
 import { VivViewerLayer, StaticImageLayer } from './layers';
 import VivViewer from './VivViewer';
+import VivViewerOverview from './VivViewerOverview';
 import { createTiffPyramid, createZarrPyramid, ZarrLoader } from './loaders';
 
 export {
   VivViewerLayer,
   VivViewer,
+  VivViewerOverview,
```

Comment:
I probably wouldn't export this (see comment in `App.js`), unless you think it would be useful! IMO it wouldn't really make sense to use this component on its own (because the bounding box data-binding isn't super straightforward to set up), and if you wanted to render a full, non-zoomable image with viv I'd just use the Static layer and turn off interactions.

Reply:
Ok.